brave/browser-laptop
# Improves verified flow

Resolves #13116

## Submitter Checklist:


## Test Plan:
1) clean profile, enable payments and add some publishers to the table (add at least one verified publisher)
2) close browser
3) edit session file where you set value 1 to all `verifiedTimestamp` and `publisherTimestamp`. Also change status of verified publisher to false
4) restart browser, visit one random site and check if verified publisher is verified again

## Reviewer Checklist:


Tests
